cmd/cmd.go

```diff
@@ -17,8 +18,12 @@ func isTest(path string) bool {
 
 	file := filepath.Base(path)
 	name := file[:len(file)-len(ext)]
-	if name == "test" {
+	if name == "test" || name == "spec" {
```

Comment:
I like that idea, seems much safer.
On Jan 21, 2016 2:37 PM, "Katrina Owen" <notifications@github.com> wrote:

> In cmd/cmd.go
> <https://github.com/exercism/cli/pull/273#discussion_r50451141>:
>
> > @@ -17,8 +18,12 @@ func isTest(path string) bool {
> >
> >  	file := filepath.Base(path)
> >  	name := file[:len(file)-len(ext)]
> > -	if name == "test" {
> > +	if name == "test" || name == "spec" {
>
> perl doesn't have test or spec in the name, for what it's worth. Languages
> that have a different pattern for the test files have this defined in the
> language config, so we could expose it via the API (and perhaps write it to
> a dotfile somewhere that we can load and check.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/exercism/cli/pull/273/files#r50451141>.
>